From 01ef786ac0738139221a9fefc4e0cdd89b622b55 Mon Sep 17 00:00:00 2001
Message-Id: <01ef786ac0738139221a9fefc4e0cdd89b622b55.1431284209.git.chang-joon.lee@intel.com>
In-Reply-To: <89460f3fc3d6c0659d44897a6adec3c3fecc5564.1431284209.git.chang-joon.lee@intel.com>
References: <89460f3fc3d6c0659d44897a6adec3c3fecc5564.1431284209.git.chang-joon.lee@intel.com>
From: Ramalingam C <ramalingam.c@intel.com>
Date: Sun, 12 Apr 2015 16:40:59 +0530
Subject: [PATCH 62/68] FOR_UPSTREAM [VPG]: drm/i915: Initializing DRRS for
 all connectors

For all the connectors drrs init is invoked. This function will
initialize the drrs for the connectors those support DRRS.

Issue: GMINL-8115
Change-Id: I1783e81db6e60db215e7290f8ebe9ebe1dd3ae9c
Signed-off-by: Ramalingam C <ramalingam.c@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 6e31792..aaa75e7 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -13683,6 +13683,7 @@ void intel_modeset_init(struct drm_device *dev)
 	int sprite, ret;
 	enum pipe pipe;
 	struct intel_crtc *crtc;
+	struct intel_connector *connector;
 
 	drm_mode_config_init(dev);
 
@@ -13746,6 +13747,11 @@ void intel_modeset_init(struct drm_device *dev)
 	i915_disable_vga(dev);
 	intel_setup_outputs(dev);
 
+	list_for_each_entry(connector, &dev->mode_config.connector_list,
+								base.head) {
+		intel_drrs_init(dev, connector, connector->panel.fixed_mode);
+	}
+
 	/* Just in case the BIOS is doing something questionable. */
 	intel_disable_fbc(dev);
 
-- 
1.7.9.5

